bot/exts/holidays/holidayreact.py

```diff
@@ -61,7 +61,7 @@ class Holiday(NamedTuple):
     }
 )
 Christmas = Holiday([Month.DECEMBER], {
-    "christmas tree": Trigger(r"\b((christ|x)mas|tree)\b", ["\U0001F384"]),
+    "christmas tree": Trigger(r"\b(christ|x)mas\s?(tree(s)?)?\b", ["\U0001F384"]),
```

Comment:
the parens around the `s` for pluralizing "tree" are unnecessary, but it might be slightly more readable with them